tests/_support/Step/Acceptance/AbstractStep.php

```diff
@@ -12,8 +12,12 @@
 
 /**
  * Class AbstractStep
+<<<<<<< HEAD
```

Comment:
@hoanghairedweb seem like we have the conflict . please help me check again. thanks